Update words.md - stilistische Änderungsvorschläge L302 - 349 (#21)
TQueV/covid-19: words/words.md

		<iframe src="sim?stage=int-2a&format=calc" width="285" height="260"></iframe>
</div>

Lass uns nun simulieren, was mit einer COVID-19-Epidemie passiert, wenn wir ab März 2020 das Händewaschen verstärkt hätten, aber nur *wenig* Abstand gehalten hätten - so dass R niedriger, aber immer noch größer als 1 ist:

<div class="sim">
		<iframe src="sim?stage=int-2&format=lines" width="800" height="540"></iframe>
</div>

Drei Anmerkungen:

> 1. Dies *verringert* die Gesamtzahl der Fälle! **Selbst wenn R < 1 nicht erreicht wird, rettet die Reduzierung von R immer noch Leben, indem sie den 'Überhang' bis zur Herdenimmunität reduziert.** Viele Menschen denken, dass mit dem Abflachen der Kurve die Fälle zeitlich gestreckt werden, ohne aber die Gesamtzahl zu reduzieren. Doch das steht im Widerspruch zu *jedem* grundlegenden Epidemiologie-Modell. Aber weil in den Nachrichten "80%+ werden infiziert" als unvermeidlich dargestellt wurde, kam es zu der weitläufigen Annahme, dass die Gesamtzahl der Fälle in jedem Fall gleich bleiben würde. *Seufz.*

Reviewer: ```suggestion
1. Dies *verringert* die Gesamtzahl der Fälle! **Selbst wenn R < 1 nicht erreicht wird, rettet die Reduzierung von R immer noch Leben, indem sie die 'Überschreitung' über die Herdenimmunität hinaus reduziert.** Viele Menschen denken, dass mit dem Abflachen der Kurve die Fälle zeitlich gestreckt werden, ohne aber die Gesamtzahl zu reduzieren. Doch das steht im Widerspruch zu *jedem* grundlegenden Epidemiologie-Modell. Aber weil in den Nachrichten "80%+ werden infiziert" als unvermeidlich dargestellt wurde, kam es zu der weitläufigen Annahme, dass die Gesamtzahl der Fälle in jedem Fall gleich bleiben würde. *Seufz.*
```
1. Woher kommt "overshoot" = "Überhang"?
2. "_above_ herd immunity" -> "_bis_ zur Herdenimmunität"?, für mich klar "über die Herdenimmunität (hinaus)". Verstehe das "bis" nicht.

Reviewer 2: Damit wird der Unterschied zwischen Fallzahl und Herdenimmunitätsschwellwert gemeint - treffender ist natürlich "overshoot" - aber "Überhang" klingt hier auch merkwürdig - wie wäre es einfach mit "Überschuss"?

Reviewer 3: Ja, Überschuss finde ich gut.

2. Aufgrund der zusätzlichen Interventionen erreichen die aktuellen Fallzahlen ihren Höhepunkt *bevor* die Herdenimmunität erreicht wird. Tatsächlich schießt in dieser Simulation die Gesamtzahl der Fälle nur *ein kleines bisschen* über die Herdenimmunität hinaus - der Plan von Großbritannien! An diesem Punkt, R < 1, kann man alle anderen Interventionen zurückfahren, und COVID-19 bleibt eingedämmt! Es gibt nur ein Problem ...

3. Es fehlen die Betten auf den Intensivstationen. Und das über mehrere Monate. (und dabei haben wir die Betten auf den Intensivstationen für diese Simulation ja *bereits verdreifacht*)

Das war das andere Ergebnis des Imperial-College-Berichts vom 16. März, der Großbritannien davon überzeugte, seinen ursprünglichen Plan aufzugeben. Jeder Versuch einer **Milderung** (R reduzieren, aber R > 1) wird scheitern. Der einzige Ausweg ist **Unterdrückung** (R reduzieren, so dass R < 1).
![](pics/mitigation_vs_suppression.png)

Das heißt: Es reicht nicht, die Kurve einfach "abzuflachen". Die Kurve muss *zerstampft* werden. Zum Beispiel mit ...

###Szenario 2: Monatelanger Lockdown

> Mal sehen, was passiert, wenn wir die Kurve mit einen 5-monatigen Lockdown *zerstampfen*, die <icon i></icon> auf fast null reduzieren und dann endlich – *endlich* – wieder zu unserem normalen Leben zurückkehren:

Reviewer: ```suggestion
Mal sehen, was passiert, wenn wir die Kurve mit einen 5-monatigen Lockdown *zermalmen*, die <icon i></icon> auf fast null reduzieren und dann endlich – *endlich* – wieder zu unserem normalen Leben zurückkehren:
```
Oder evtl. "zerdrücken". Bei "zerstampfen" muss ich nur an Kartoffelbrei denken 😅.

Reviewer: Finde aber bildlich bei der Kurve "stampfen" oder "drücken" eigentlich ganz passend. Seht das mal als Alternativvorschlag ohne starke Eigenmeinung. Hab es auch sonst nirgendwo abgeändert.

Reviewer 3: vielleicht sonst auch zerschmettern? Aber kann mich auch mit den anderen Vorschlägen anfreunden.

Reviewer 2: Ich würde drücken auch favorisieren.

Author: Ja, drücken finde ich auch gut.

Drei Anmerkungen:

1. Dies *verringert* die Gesamtzahl der Fälle! **Selbst wenn R < 1 nicht erreicht wird, rettet die Reduzierung von R immer noch Leben, indem sie den 'Überschuss' bis zur Herdenimmunität reduziert.** Viele Menschen denken, dass mit dem Abflachen der Kurve die Fälle zeitlich gestreckt werden, ohne aber die Gesamtzahl zu reduzieren. Doch das steht im Widerspruch zu *jedem* grundlegenden Epidemiologie-Modell. Aber weil in den Nachrichten "80%+ werden infiziert" als unvermeidlich dargestellt wurde, kam es zu der weitläufigen Annahme, dass die Gesamtzahl der Fälle in jedem Fall gleich bleiben würde. *Seufz.*

> 2. Aufgrund der zusätzlichen Maßnahmen erreichen die aktuellen Fälle ihren Höhepunkt *bevor* die Herdenimmunität erreicht wird. Tatsächlich schießt in dieser Simulation die Gesamtzahl der Fälle nur *ein kleines bisschen* über die Herdenimmunität hinaus - der Plan des Vereinigten Königreichs! An diesem Punkt, R < 1, kann man alle anderen Maßnahmen loslassen, und COVID-19 bleibt eingedämmt! Es gibt nur ein Problem ...

Reviewer: ```suggestion
2. Aufgrund der zusätzlichen Maßnahmen erreichen die aktuellen Fallzahlen ihren Höhepunkt *bevor* die Herdenimmunität erreicht wird. Tatsächlich schießt in dieser Simulation die Gesamtzahl der Fälle nur *ein kleines bisschen* über die Herdenimmunität hinaus - der Plan des Vereinigten Königreichs! An diesem Punkt, R < 1, kann man alle anderen Maßnahmen loslassen, und COVID-19 bleibt eingedämmt! Es gibt nur ein Problem ...
```
Fand "Fallzahlen" hier passender.

> 2. Aufgrund der zusätzlichen Maßnahmen erreichen die aktuellen Fälle ihren Höhepunkt *bevor* die Herdenimmunität erreicht wird. Tatsächlich schießt in dieser Simulation die Gesamtzahl der Fälle nur *ein kleines bisschen* über die Herdenimmunität hinaus - der Plan des Vereinigten Königreichs! An diesem Punkt, R < 1, kann man alle anderen Maßnahmen loslassen, und COVID-19 bleibt eingedämmt! Es gibt nur ein Problem ...

Reviewer: ```suggestion
2. Aufgrund der zusätzlichen Maßnahmen erreichen die aktuellen Fälle ihren Höhepunkt *bevor* die Herdenimmunität erreicht wird. Tatsächlich schießt in dieser Simulation die Gesamtzahl der Fälle nur *ein kleines bisschen* über die Herdenimmunität hinaus - der Plan des Vereinigten Königreichs! An diesem Punkt, R < 1, kann man alle anderen Maßnahmen loslassen, und COVID-19 bleibt eingedämmt! Es gibt nur ein Problem ...
```
Fand "Maßnahmen zurückfahren" oder von mir aus auch "runterfahren" geeigneter. Ist "loslassen" so gängig? Scheint mir aus dem Englischen von "let go of sth." zu kommen, aber evtl. bin ich da nicht auf der Höhe des aktuellen Sprachgebrauchs.

Diese Option wurde zuerst im Bericht des Imperial College vom 16. März und später erneut in einem Harvard-Papier vorgeschlagen [^lockdown_harvard].

[^lockdown_harvard]: In Abwesenheit anderer Interventionen ist das Überschreiten der Intensivpflege-Kapazitäten eine Schlüsselgröße für den Erfolg des social distancing. Um dies zu vermeiden, kann längeres oder zeitweiliges social distancing bis ins Jahr 2022 erforderlich sein. (übersetzt): “Absent other interventions, a key metric for the success of social distancing is whether critical care capacities are exceeded. To avoid this, prolonged or intermittent social distancing may be necessary into 2022.”[Kissler und Tedijanto et al] (https://science.sciencemag.org/content/early/2020/04/14/science.abb5793)

> **Hier ist eine Simulation für diese Option:** (Spiele zunächst die Aufzeichnung ab. Danach kannst du versuchen, deinen *eigenen* Lockdown-Plan zu simulieren, indem du den Schieberegler änderst ** während* die Simulation läuft! Du kannst dabei die Simulation unterbrechen, fortsetzen und die Simulationsgeschwindigkeit ändern)

Reviewer: ```suggestion
**Hier ist eine Simulation für diese Option:** (Spiele zunächst die Aufzeichnung ab. Danach kannst du versuchen, deinen *eigenen* Lockdown-Plan zu simulieren, indem du den Schieberegler änderst *während* die Simulation läuft! Du kannst dabei die Simulation unterbrechen, fortsetzen und die Simulationsgeschwindigkeit ändern)
```
nur nen Formatierungsfehler, hab's mit dem Original abgeglichen

Diese Option wurde zuerst im Bericht des Imperial College vom 16. März und später erneut in einem Harvard-Papier vorgeschlagen [^lockdown_harvard].

> [^lockdown_harvard]: In Abwesenheit anderer Maßnahmen ist eine Schlüsselmetrik für den Erfolg der sozialen Distanzierung, ob die Kapazitäten der Intensivpflege überschritten werden. Um dies zu vermeiden, kann eine längere oder zeitweilige soziale Distanzierung bis ins Jahr 2022 erforderlich sein. [Kissler und Tedijanto et al] (https://science.sciencemag.org/content/early/2020/04/14/science.abb5793)

Reviewer: ```suggestion
[^lockdown_harvard]: In Abwesenheit anderer Maßnahmen ist eine Schlüsselmetrik für den Erfolg der sozialen Distanzierung, ob die Kapazitäten der Intensivstationen überschritten werden. Um dies zu vermeiden, kann eine längere oder periodische räumliche Distanzierung bis ins Jahr 2022 erforderlich sein. (übersetzt): “Absent other interventions, a key metric for the success of social distancing is whether critical care capacities are exceeded. To avoid this, prolonged or intermittent social distancing may be necessary into 2022.” [Kissler und Tedijanto et al] (https://science.sciencemag.org/content/early/2020/04/14/science.abb5793)
```
Wir hatten uns darauf geeinigt Originalzitate mit Übersetzung zu liefern. Wie Originalzitat und Übersetzung formatiert sind, würde ich nach den PRs noch mal vereinheitlichen.
Bei social distancing bin ich mir unsicher. Bei Wikipedia bekommt man da gerade "räumliche Distanzierung" was mir eigentlich besser gefällt (da es genauer beschreibt was gemacht wird) als das Original.

Reviewer: Ob "Intensivpflege" oder "Intensivstationen" ist mir eigentlich egal
